src/timer.h

```diff
@@ -33,7 +33,10 @@
 
 static inline void
 yield() {
-#if defined PL_UNIX
+#if defined DEBUG
+    // Reduce the startup log noise.
+    usleep(50000);
```

Comment:
This causes the data validation jobs to fail because it is likely that the import stacks are not collected. Manual inspection of the collected profiles doesn't show any actual regressions